cmd/2019_7/main_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/afarbos/aoc/pkg/test"
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+func TestFindMaxThrusterSignal(t *testing.T) {
+	testData := map[int][]int{
+		43210: []int{3, 15, 3, 16, 1002, 16, 10, 16, 1, 16, 15, 15, 4, 15, 99, 0, 0},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 85 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)